apps/test/unit/lib/levelbuilder/lesson-editor/programmingExpressionsEditorReduxTest.js

```diff
@@ -24,6 +25,24 @@ describe('programmingExpressionsEditorRedux reducer tests', () => {
   let initialState;
   beforeEach(() => (initialState = getInitialState()));
 
+  it('add programmingExpression', () => {
+    const nextState = programmingExpressionEditor(
+      initialState,
+      addProgrammingExpression({
+        id: 1,
+        key: 'programmingExpression-3',
+        name: 'repeatSound',
+        category: 'UI controls',
+        programmingEnvironmentName: 'applab'
+      })
+    );
+    assert.deepEqual(nextState.map(r => r.key), [
```

Comment:
we should also have an assertion verifying that our state was not this prior to the action